android/build.gradle

```diff
@@ -198,11 +202,7 @@ def kotlin_version = safeExtGet('kotlinVersion', project.properties['RNGH_kotlin
 
 dependencies {
     //noinspection GradleDynamicVersion
-    if (isNewArchitectureEnabled()) {
-        implementation project(':ReactAndroid')
-    } else {
-        implementation 'com.facebook.react:react-native:+'
-    }
+    implementation 'com.facebook.react:react-native:+' // TODO
```

Comment:
```suggestion
    if (REACT_NATIVE_MINOR_VERSION >= 71) {
         implementation "com.facebook.react:react-android" // version substituted by RNGP
    } else {
        implementation "com.facebook.react:react-native:+" // From node_modules
    }
```

Reply:
Done in https://github.com/software-mansion/react-native-gesture-handler/pull/2313/commits/ae8149a057b300982906209470e214ecfd6c8e4e